bazel/repository_locations.bzl

```diff
@@ -842,7 +842,7 @@ REPOSITORY_LOCATIONS_SPEC = dict(
             "envoy.filters.network.wasm",
             "envoy.stat_sinks.wasm",
         ],
-        release_date = "2020-10-16",
+        release_date = "2020-10-20",
```

Comment:
UTC for the commit is `2020-10-21`. From source at https://github.com/proxy-wasm/proxy-wasm-cpp-host/commit/2d4bfe9c29fc957c3c4e81e6575773801eedade1
```html
<relative-time datetime="2020-10-21T02:11:41Z" class="no-wrap" title="Oct 20, 2020, 7:11 PM PDT">16 hours ago</relative-time>
```

Reply:
shouldn't the dependencies CI check for this? cc @htuch

Reply:
Yeah, I wonder if there is a bug in the script where we're accidentally switching into local timezone, it's supposed to be UTC. I'll look into this. Since this is only 2 hours away from midnight, that's my best guess.

Reply:
Done.